core/src/main/java/io/questdb/griffin/engine/groupby/vect/AvgShortVectorAggregateFunction.java

```diff
@@ -74,11 +74,11 @@ public void aggregate(long address, long addressSize, int columnSizeHint, int wo
     }
 
     @Override
-    public boolean aggregate(long pRosti, long keyAddress, long valueAddress, long valueAddressSize, int columnSizeShr, int workerId) {
```

Comment:
API clean up, we really used to pass two parameters just to get to the "row count"